docs/en/faq.md

```diff
@@ -123,6 +123,64 @@ You can accelerate report loading by converting QuickSight datasets to SPICE mod
 
+
+### How to implement a dedicated Redshift for Analytics Studio?
+
+Redshift supports [sharing data across different Redshift clusters][redshift-share-data], allowing you to use a dedicated Redshift cluster for Analytics Studio to achieve better query performance and cost optimization.
+
+Before implementing Amazon Redshift data sharing, please note the following:
+
+- You can share data between cluster types, as well as between configured clusters and serverless ones.
+- Only **RA3** type clusters and **Redshift serverless** support data sharing.
+
+Taking Redshift serverless as an example of data sharing, follow these operational steps:
+
+1. [Create a Redshift serverless][serverless-console-workflows] as the data consumer.
+2. Run SQL in the producer Redshift database (The project database configured in the Clickstream solution) to create a data share and grant consumer permissions:
+    ```sql
+    -- Create Datashare
+
+    CREATE DATASHARE bi SET PUBLICACCESSIBLE FALSE;
+    ALTER DATASHARE bi ADD SCHEMA <schema>;
+    ALTER DATASHARE bi ADD ALL TABLES IN SCHEMA <schema>;
+
+    -- Grant the Datashare to the consumer Redshift.
+
+    GRANT USAGE ON DATASHARE bi TO NAMESPACE '<target namespace id>';
+    ```
+    Replace `<schema>` with the schema you want to share, and `<target namespace id>` with the consumer Redshift serverless namespace ID.
+3. Run SQL in the consumer Redshift database:
+    ```sql
+    -- Create Datashare 
+
+    CREATE DATASHARE <new database name> WITH PERMISSIONS FROM DATASHARE bi OF NAMESPACE '<source namespace id>';
+   
+    -- Create bi user
+    
+    CREATE USER bi_user PASSWORD '<strong password>';
+    GRANT USAGE ON DATABASE "<new database name>" TO bi_user;
+    GRANT USAGE ON SCHEMA "<new database name>"."<schema>" TO bi_user;
+    GRANT SELECT ON ALL TABLES IN SCHEMA "<new database name>"."<schema>" TO bi_user;
+
+    -- Test bi_user permission (optional)
+    SET SESSION AUTHORIZATION bi_user;
+    SELECT CURRENT_USER;
+    SELECT * FROM "<new database name>"."<schema>"."event_v2" limit 1;
+    ```
+    Replace `<new database name>` with the database name in the consumer Redshift (it can be different from the original database name), and `<source namespace id>` with the producer Redshift serverless namespace ID.
+4. Create a new secret for the BI user in Secrets Manager, specifying the value as plaintext like below:
+   ```json
+   {"username":"bi_user","password":"<strong password>"}
+   ```
+   The key name should be like: `/clickstream/reporting/user/bi_user`.
+5. Update the reporting stack to use the consumer Redshift:
+    - **Redshift Endpoint Url** (Required): Consumer Redshift access endpoint
+    - **Redshift Default database name** (Required): `dev`
+    - **Redshift Database Name** (Required): `<new database name>`
+    - **Parameter Key Name** (Required): `/clickstream/reporting/user/bi_user`
```

Comment:
use <key name> instead of "/clickstream/reporting/user/bi_user" ?

Reply:
5. Go to Cloudformation in AWS console, update the reporting stack to use the consumer Redshift: